scopes/workspace/workspace/workspace.ts

```diff
@@ -1191,6 +1189,13 @@ export class Workspace implements ComponentFactory {
     this.clearCache();
   }
 
+  getLocalDir(component: ConsumerComponent | Component) {
```

Comment:
change the name, this name is confusing. I might see it and think that it will give me the component root dir in the workspace.
This is an external API so the name is important

Reply:
Changed the name to `getComponentPackagePath` and made it private as it's only used in workspace now (I can even remove this change if you want)